hack/convert-output-to-tsv.py

```diff
@@ -0,0 +1,51 @@
+# A script to convert transformed data into TSV files
+
+import json
+import csv
+from os import listdir
+import sys
+
+if (len(sys.argv) < 3):
+    print "Please provide the input directory and output directory as arguments!"
+
+schema_dir = '../schema/src/main/jade-tables/'
+input_dir = sys.argv[1]
+output_dir = sys.argv[2]
+table_names = listdir(input_dir) 
+pk_prefix = 'entity:'
+
+for table_name in table_names:
+    # get set of all keys/column names from the schema file
+    column_list = []
+    primary_key_list = []
+    with open(schema_dir + table_name + '.table.json') as schema_file:
+        schema = json.load(schema_file)
+        for column in schema['columns']:
+            column_name = column['name']
+            if (column.get('type') == 'primary_key'):
+                primary_key_list.append(column_name)
+                column_list.append(pk_prefix + column_name)
+            else: 
+                column_list.append(column_name)
```

Comment:
This loop isn't going to work any more now that we've scattered columns across multiple files.

Instead of collecting column names from the schemas, could we build up a `set()` of keys found while adding data to `obj_list` below?

Reply:
Oh, true... And yeah, that sounds like a good plan.